Tribler/Test/Core/Modules/RestApi/test_torrentinfo_endpoint.py

```diff
@@ -52,10 +51,9 @@ def verify_valid_dict(data):
             # To fix it, we must switch to some encoding scheme that is able to encode and decode raw binary
             # fields in the dicts.
             # However, for this works fine at the moment because we never use pieces data in the GUI.
-            #self.assertTrue(TorrentDef.load_from_dict(metainfo_dict))
+            # self.assertTrue(TorrentDef.load_from_dict(metainfo_dict))
```

Comment:
Commented-out code, remove or uncomment.

Reply:
Have you read the FIXME above?

Reply:
Oh sorry, I couldn't see the FIXME in the diff, nvm.